### DISCUSSION TITLE
Orientation details are not available

### DISCUSSION THREAD

**Discussion by gnanads** [mindee/doctr#971] [Q&A] [answered]
Hi Team,

When I exported the ocr_predictor results, I found orientation and Language information are missing.

Below is the code snippet:

```
model = ocr_predictor('linknet_resnet18_rotation','crnn_vgg16_bn',pretrained=True,assume_straight_pages=False, preserve_aspect_ratio=True)
result = model([image])
export = result.export()
print(export)
```

Result json:
{'page_idx': 0, 'dimensions': (912, 617), 'orientation': {'value': None, 'confidence': None}, 'language': {'value': None, 'confidence': None}

**Comment by frgfm** [comment, marked as answer]
Hello @gnanads :wave: 

Yes you are completely correct, we haven't updated the JSON export accordingly! We'll try to fix this shortly :ok_hand: 
By the way, this is more of an `issue/bug` than a discussion :) Would you mind if we convert this to an issue?

**Reply by gnanads** [reply]
Hi @frgfm 

Thanks for letting me know. I thought I was using wrong set of models and hence the results are not populated while exporting it as Json. I have create a new issue for this.
